frontend/src/pages/Dashboard/data/reducer.ts

```diff
@@ -50,6 +50,15 @@ const dashboardReducer = (
           : state.totalWordsLearned,
         error: undefined,
       };
+    case ActionTypes.GET_USER_SESSION_DATA_SUCCESS:
+      return {
+        ...state,
+        students: undefined,
+        user: undefined,
+        totalWordsLearned: undefined,
+        error: undefined,
```

Comment:
When we get stats, should we throw away all of the other information we have? I tend to think we should keep it, since it will make it quicker if you click the back button.